assets/css/translation.css

```diff
@@ -126,3 +126,159 @@ html[dir='rtl'] #srcLanguages {
     padding: 0 15px;
     position: relative;
 }
+
+#navcontainerPosition {
+    position: relative;
+}
+
+#apertiumLogo {
+    margin-bottom: -.25em;
+}
+
+#divPullRightHidden {
+    width: 35%;
+}
+
+#faFaGlobe {
+    padding: 5px 5px 0 0;
+}
+
+#srcLangSelect {
+    display: inline-block;
+}
+
+#dstLangSelect {
+    display: inline-block;
+}
+
+#cancelTranslateWebpage {
+    display: none;
+}
+
+#translateDoc {
+    margin-top: 1em;
+}
+
+#fileDropBackdrop {
+    display: none;
+    z-index: 1000;
+}
+
+#divforDropDocument {
+    left: 35%;
+    position: absolute;
+    top: 35%;
+    text-align: center;
+}
+
+#headerForDropDocument {
+    font-weight: bold;
+
+}
+#fileDropMask {
+    opacity: 0;
+    z-index: 10000;
+}
+#docTranslation {
+    display: none;
+}
+#fileName {
+    display: none;
+}
+
+#divForUploadError {
+    min-height: 50px;
+    margin-top: 1em;
+}
+
+#uploadError {
+    display: none;
+}
+.progress {
```

Comment:
Blank line between rules (everywhere :smiley:). Also, this is used only once?

Reply:
ya had to use class as this was uneffected i.e, never progress class is used else where but here.